mw_jwt_test.go

```diff
@@ -86,39 +87,84 @@ func createJWTSessionWithRSAWithPolicy(policyID string) *user.SessionState {
 	return session
 }
 
-// JWTSessionHMAC
+type JwtCreator func() *user.SessionState
+
+var createHmacJWKSession = func() *user.SessionState { return createJWTSession() }
```

Comment:
I am confused why do we need this new funcs and not to use just names `createJWTSession` and `createJWTSessionWithRSA` - I think they qualify to be type of `JwtCreator`

Reply:
fixed